//usr/lib/i386-linux-gnu/libharfbuzz.so.0: Warnung: undefinierter Verweis auf »FT_Get_Var_Blend_Coordinates«

##### Issue type
<!-- Pick one below and delete others -->
 - Build issue
 
##### Minetest version
<!--
Paste Minetest version between quotes below
If you are on a devel version, please add git commit hash
You can use `minetest --version` to find it.
-->
```
actual Git Version from today
```

##### OS / Hardware
<!-- General information about your hardware and operating system -->
Operating system: Kubuntu 18.04 LTS (32Bit)
CPU: Intel

<!-- For graphical issues only -->
GPU model:
OpenGL version:

##### Summary
<!-- Describe your problem here -->
//usr/lib/i386-linux-gnu/libharfbuzz.so.0: Warnung: undefinierter Verweis auf »FT_Get_Var_Blend_Coordinates«
collect2: error: ld returned 1 exit status
src/CMakeFiles/minetest.dir/build.make:7108: recipe for target 'bin/minetest' failed
make[2]: *** [bin/minetest] Error 1
CMakeFiles/Makefile2:211: recipe for target 'src/CMakeFiles/minetest.dir/all' failed
make[1]: *** [src/CMakeFiles/minetest.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2


##### Steps to reproduce
<!-- For bug reports or build issues, explain how the problem happened -->
after updating my Kubuntu 16.04 to 18.04 I can´t compile anymore.

--- a/doc/lua_api.txt
+++ b/doc/lua_api.txt
@@ -2037,11 +2037,15 @@ Elements
 
 * Show an inventory image of registered item/node
 
-### `bgcolor[<color>;<fullscreen>]`
+### `bgcolor[<color>]`
 
 * Sets background color of formspec as `ColorString`
-* If `true`, the background color is drawn fullscreen (does not affect the size
-  of the formspec).
+
+### `bgcolor[<color>;<fullscreen>]`
+
+* If `color` is a valid `ColorString`, the fullscreen background color
+  is set to `color`.
+* If `fullscreen` is a true value, the fullscreen background color is drawn.
 
 ### `background[<X>,<Y>;<W>,<H>;<texture name>]`
 

--- a/src/gui/guiFormSpecMenu.cpp
+++ b/src/gui/guiFormSpecMenu.cpp
@@ -1795,17 +1795,17 @@ void GUIFormSpecMenu::parseBox(parserData* data, const std::string &element)
 	errorstream<< "Invalid Box element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
-void GUIFormSpecMenu::parseBackgroundColor(parserData* data, const std::string &element)
+void GUIFormSpecMenu::parseBackgroundColor(parserData *data, const std::string &element)
 {
 	std::vector<std::string> parts = split(element,';');
 
 	if (((parts.size() == 1) || (parts.size() == 2)) ||
 			((parts.size() > 2) && (m_formspec_version > FORMSPEC_API_VERSION))) {
-		parseColorString(parts[0], m_bgcolor, false);
-
-		if (parts.size() == 2) {
-			std::string fullscreen = parts[1];
-			m_bgfullscreen = is_yes(fullscreen);
+		if (parts.size() == 1) {
+			parseColorString(parts[0], m_bgcolor, false);
+		} else if (parts.size() == 2) {
+			parseColorString(parts[0], m_fullscreen_bgcolor, false);
+			m_bgfullscreen = is_yes(parts[1]);
 		}
 
 		return;
@@ -2775,8 +2775,7 @@ void GUIFormSpecMenu::drawMenu()
 
 	if (m_bgfullscreen)
 		driver->draw2DRectangle(m_fullscreen_bgcolor, allbg, &allbg);
-	else
-		driver->draw2DRectangle(m_bgcolor, AbsoluteRect, &AbsoluteClippingRect);
+	driver->draw2DRectangle(m_bgcolor, AbsoluteRect, &AbsoluteClippingRect);
 
 	m_tooltip_element->setVisible(false);
 

